rmarkdowns/GISSR_W3_Clemens_Mayer.Rmd

```diff
@@ -0,0 +1,54 @@
+```{r}
+available_tags("landuse")
+```
+
+```{r}
+bb = getbb("Leiden")
+#Tried Eauripik and Woleai again, but schools and most other features are not mapped.Tried Yap also, but elements are only very partially mapped.
```

Comment:
Thanks for trying! Weʻll have to look for other data sources; and this is just a general problem we run into on many Pacific Islands. One thing I want to try to work on in the future is building community mapping groups to fill in some of these persistent gaps...